src/css/custom.css

```diff
@@ -15,6 +15,8 @@
   --ifm-color-primary-lightest: #f871b4;
   --ifm-code-font-size: 95%;
   --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.1);
+
+  --ifm-navbar-height: inherit;
```

Comment:
This allows the header to grow to accommodate larger labels

Without this when a label wraps to two lines, it gets cut off by the explicit height of the navbar